packages/shared/src/components/ShareBar.tsx

```diff
@@ -42,53 +44,70 @@ export default function ShareBar({ post }: { post: Post }): ReactElement {
     onClick(ShareProvider.CopyLink);
   };
 
+  const onShareToSquad = (squad: Squad) => {
+    trackEvent(postAnalyticsEvent(AnalyticsEvent.StartShareToSquad, post));
+    openModal({
+      type: LazyModal.CreateSharedPost,
+      props: {
+        squad,
+        preview: post,
+        onSharedSuccessfully: () => {
+          return trackEvent(
+            postAnalyticsEvent(AnalyticsEvent.ShareToSquad, post),
+          );
+        },
+      },
+    });
+  };
+
   return (
     <WidgetContainer className="hidden laptop:flex flex-col p-3">
-      <p className="typo-callout text-theme-label-tertiary">
-        Would you recommend this post?
+      <p className="mb-4 typo-callout text-theme-label-tertiary">
+        One-click share
```

Comment:
@sshanzel I know design says this, but we decided afterwords it should remain the same, sorry it's a bit messy in sketch.

Reply:
Oh, cool. I shall revert this.